lib/trivial.nix

```diff
@@ -1,17 +1,44 @@
 rec {
 
-  # Identity function.
+  /* The identity function
+     For when you need a function that does “nothing”.
+
+     Type: id :: a -> a
```

Comment:
It’s a style I copied from `lib/attrsets`.
Doing that in a consistent manner will greatly help with generating docs.